cmd/node/node.go

```diff
@@ -511,7 +512,8 @@ func (app *SpacemeshApp) Start(cmd *cobra.Command, args []string) {
 
 	app.unregisterOracle = func() { oracleClient.Unregister(true, app.edSgn.PublicKey().String()) }
 
-	rng := BLS381.DefaultSeed()
+	rng := amcl.NewRAND()
+	rng.Seed(10, app.edSgn.Sign([]byte{1, 2, 3, 4, 5, 6, 7, 8, 9, 10}))
```

Comment:
add comment, consider having a different byte array to sign (maybe the bls pub key? or any other pub key)

Reply:
Fixed